test/test_json.rb

```diff
@@ -89,4 +90,17 @@ def test_generate_raises_error_for_unexpected_value_type
     end
     assert_equal 'Unexpected value of type Regexp', ex.message
   end
+
+  private
+
+  def assert_puma_json_generates_string(expected_output, value_to_serialize, expected_roundtrip: value_to_serialize)
```

Comment:
I've never seen this before. Does `value_to_serialize` refer to the value that was passed in as the second argument?

Reply:
It seems to work that way, but I couldn't find that behavior being explicitly supported in any official Ruby documentation, so I made it a little more clear in 0bb2df3